Before: b47a25500cb2540489b9f3498f62edfb607e4c1118e8ab5ef82b2f949c652ce4
After: f85aa0d0452b467e63f363f2e35479e0b3a004618153cef1f87541c6e4d3bec3
Patch: 3700569da2f303f06529cf9ba74dd8a3b00e4e75532122a8d2c96e857312cd55
--- .simplecov	2012-07-22 20:44:49.385583281 +1000
+++ .simplecov	2012-07-22 21:28:02.226874697 +1000
@@ -4,11 +4,8 @@
   # add_group 'Models', 'app/model'
   # add_group 'Plugins', '(app|lib)/plugins'
 
-  # Remove test code from coverage
+  # Remove coverage for test code
   add_filter 'test'
-
-  # Add files that don't get required to coverage too
-  add_files_to_coverage_at_exit '{app,lib}/**/*.rb'
 end
 
-# vim:ft=ruby et ts=2 sw=2:
+# vim: ft=ruby et ts=2 sw=2 :
--- Gemfile.corvid	2012-07-22 20:44:49.382249972 +1000
+++ Gemfile.corvid	2012-07-22 21:28:02.230208006 +1000
@@ -1,5 +1,6 @@
-# vim:ft=ruby et ts=2 sw=2:
+# vim: ft=ruby et ts=2 sw=2 :
 header "# This file is generated by corvid update:deps"
+footer "# v\im: ft=ruby et ts=2 sw=2 :"
 
 if options[:use_corvid_gem]
   gem 'corvid', ">= #{::Corvid::VERSION}"
@@ -10,8 +11,7 @@
 
 group :test do
   gem 'ci_reporter', require: false
-  #gem 'simplecov', require: false
-  gem 'simplecov', require: false, git: 'git://github.com/japgolly/simplecov.git', branch: 'nan_fix'
+  gem 'simplecov', require: false
 end if Dir.exists?('test')
 
 group :test_unit do
